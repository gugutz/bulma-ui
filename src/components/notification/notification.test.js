import React from 'react'
import renderer from 'react-test-renderer'
import Notification from './notification'

describe('<Notification /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Notification>
          <div>Some content</div>
        </Notification>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isPrimary prop', () => {
    const tree = renderer
      .create(
        <Notification isPrimary>
          <div>Some content</div>
        </Notification>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isLink prop', () => {
    const tree = renderer
      .create(
        <Notification isLink>
          <div>Some content</div>
        </Notification>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isInfo prop', () => {
    const tree = renderer
      .create(
        <Notification isInfo>
          <div>Some content</div>
        </Notification>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isSuccess prop', () => {
    const tree = renderer
      .create(
        <Notification isSuccess>
          <div>Some content</div>
        </Notification>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isWarning prop', () => {
    const tree = renderer
      .create(
        <Notification isWarning>
          <div>Some content</div>
        </Notification>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isDanger prop', () => {
    const tree = renderer
      .create(
        <Notification isWarning>
          <div>Some content</div>
        </Notification>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
