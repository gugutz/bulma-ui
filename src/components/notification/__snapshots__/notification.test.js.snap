// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notification /> component should render correctly 1`] = `
<div
  className="notification"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Notification /> component should support isDanger prop 1`] = `
<div
  className="notification is-warning"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Notification /> component should support isInfo prop 1`] = `
<div
  className="notification is-info"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Notification /> component should support isLink prop 1`] = `
<div
  className="notification is-link"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Notification /> component should support isPrimary prop 1`] = `
<div
  className="notification is-primary"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Notification /> component should support isSuccess prop 1`] = `
<div
  className="notification is-success"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Notification /> component should support isWarning prop 1`] = `
<div
  className="notification is-warning"
>
  <div>
    Some content
  </div>
</div>
`;
